index.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
 		'no-bind': require( './rules/no-bind' ),
 		'no-box-model': require( './rules/no-box-model' ),
 		'no-browser': require( './rules/no-browser' ),
+		'no-camelcase': require( './rules/no-camelcase' ),
```

Comment:
The rule name should be no-camel-case, as isFunction -> is-function, parseHTML -> parse-html etc.

Reply:
Renamed